### DISCUSSION TITLE
Property "features" not displayed when logging "event"

### DISCUSSION THREAD

**Discussion by nikmatswace** [uber/react-map-gl#2085] [Q&A]
Why doesn't `features` exist as a property when logging `event` even though it is there and has a value?

<img width="413" alt="image" src="https://user-images.githubusercontent.com/79859608/208115830-ced3d9e0-061d-4738-ac2f-d78a2f12f5ae.png">
<img width="436" alt="image" src="https://user-images.githubusercontent.com/79859608/208115915-7b5a648e-115b-4bb7-b2c0-24f4ef479ee3.png">

**Comment by irakli2206** [comment]
Exact same issue
